src/auth/Fingerprint.cpp

```diff
@@ -218,15 +243,20 @@ void CFingerprint::startVerify(bool isRetry) {
     m_sDBUSState.device->callMethodAsync("VerifyStart").onInterface(DEVICE).withArguments(finger).uponReplyInvoke([this, isRetry](std::optional<sdbus::Error> e) {
         if (e) {
             Debug::log(WARN, "fprint: could not start verifying, {}", e->what());
-            if (isRetry)
-                m_sDBUSState.message = "Fingerprint auth disabled (failed to restart)";
+            if (isRetry) {
+                m_sPrompt        = "";
+                m_sFailureReason = "Fingerprint auth disabled (failed to restart)";
+            }
         } else {
             Debug::log(LOG, "fprint: started verifying");
             if (isRetry) {
                 m_sDBUSState.retries++;
-                m_sDBUSState.message = RETRY_MESSAGE;
-            } else
-                m_sDBUSState.message = m_sFingerprintReady;
+                m_sPrompt        = RETRY_MESSAGE;
```

Comment:
@PaideiaDilemma just curious, why is this the only message that is predefined as a const?

Reply:
No particular reason, and its not in style as well (Consts are without `_`). You could remove it and substitute for the string.